stacks/minstack.js

```diff
@@ -0,0 +1,56 @@
+
+function MinStack(){
```

Comment:
why not use existing implementation of stack like this https://github.com/sharadJay/datastructures-algorithms-javascript/blob/master/datastructures/Stacks/minStack.js